front/components/assistant_builder/AssistantBuilder.tsx

```diff
@@ -446,6 +447,21 @@ export default function AssistantBuilder({
       }
     }
 
+    if (builderState.actionMode === "PROCESS") {
```

Comment:
Can you also edit line 282 (search on "// Not supported in the builder yet.") 🙏🏻

Reply:
Yep adding "PROCESS" there even if no impact for now